packages/grid/x-grid/src/tests/reorder.XGrid.test.tsx

```diff
@@ -24,6 +24,25 @@ import {
 
 const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 
+function createDragOverEvent(target: ChildNode) {
+  const dragOverEvent = createEvent.dragOver(target);
+  // Safari 13 doesn't have DragEvent.
+  // RTL fallbacks to Event which doesn't allow to set these fields during initialization.
+  Object.defineProperty(dragOverEvent, 'clientX', { value: 1 });
```

Comment:
For another PR:

Add an argument to allow to change the `clientX`. We use this value to check if the user is moving to the left or right.